airflow/utils/task_group.py

```diff
@@ -95,8 +109,23 @@ def __init__(
             self.used_group_ids = self._parent_group.used_group_ids
 
         self._group_id = group_id
-        if self.group_id in self.used_group_ids:
-            raise DuplicateTaskIdFound(f"group_id '{self.group_id}' has already been added to the DAG")
+        # if given group_id already used assign suffix by incrementing largest used suffix integer
+        # Example : task_group ==> task_group__1 -> task_group__2 -> task_group__3
+        if group_id in self.used_group_ids:
+            base = re.split(r'__\d+$', group_id)[0]
+            print([(i, type(i)) for i in self.used_group_ids])
```

Comment:
We don't want a print statement here I think -- (maybe a remnant od debugging)

Reply:
ya,